conf/machine/include/rpi-default-versions.inc

```diff
@@ -1,3 +1,3 @@
 # RaspberryPi BSP default versions
 
-PREFERRED_VERSION_linux-raspberrypi ??= "4.9.%"
+PREFERRED_VERSION_linux-raspberrypi ??= "4.14.%"
```

Comment:
From the git repository 4.9 still seems to be the default branch. How did we decide on this version's preference?

Reply:
This will switch to 4.14 soon'ish, we still have couple of months for sumo release. We can track back to 4.9 if they dont switch default by then, but this will give enough testing for 4.14 from our end

Reply:
https://github.com/raspberrypi/firmware has been changed to use 4.14.y.

Reply:
WIFI/BLE comes up, userland/gstreamer works with WPE video acceleration is good so all my stringent tests seems to pass with 4.14 as of now.

Reply:
Thanks guys.